package.json

```diff
@@ -127,12 +127,13 @@
       "webpack-cli": "^4.9.2"
    },
    "dependencies": {
+      "@azure/ms-rest-azure-env": "^2.0.0",
+      "@microsoft/vscode-azext-utils": "^0.3.15",
       "download": "^8.0.0",
       "handlebars": "^4.7.7",
       "node-fetch": "^3.2.5",
       "shelljs": "^0.8.5",
       "tmp": "^0.2.1",
-      "vscode-extension-telemetry": "^0.1.7",
```

Comment:
💡 why is this PR removing the telemetry form the extension? 

Cc: @sabbour

Reply:
This is documented in the description.

> This new utility reports telemetry with the same vscode-extension-telemetry package that was previously used.

Internally the official VS Code Azure Utilities behaves the same exact way as the previously used custom reporter.